linodecli/plugins/obj.py

```diff
@@ -798,50 +819,46 @@ def call(args, context):  # pylint: disable=too-many-branches,too-many-statement
         type=str,
         help="The cluster to use for the operation",
     )
+    return parser
 
-    parsed, args = parser.parse_known_args(args)
 
-    # don't mind --no-defaults if it's there; the top-level parser already took care of it
-    try:
-        args.remove("--no-defaults")
-    except ValueError:
-        pass
+def call(
+    args: List[str], context: PluginContext
+):  # pylint: disable=too-many-branches,too-many-statements
+    """
+    This is called when the plugin is invoked
+    """
+    if not HAS_BOTO:
+        # we can't do anything - ask for an install
+        pip_version = "pip3" if sys.version[0] == 3 else "pip"
```

Comment:
Wondering if we can drop this expression now that Python 2 support is dropped.

Reply:
Yea, that's on my TODO list.